modules/gateway/peers.go

```diff
@@ -26,10 +24,37 @@ const (
 	minNodeListLen = 100
 )
 
+var (
+	// The gateway will sleep this long between incoming connections.
+	acceptInterval = func() time.Duration {
```

Comment:
does not apply to incoming rpcs either? if not, then there's no issue. Slow peer connection times isn't a concern.